pkgs/applications/audio/zynaddsubfx/default.nix

```diff
@@ -123,8 +141,15 @@ in stdenv.mkDerivation rec {
       then "https://zynaddsubfx.sourceforge.io/zyn-fusion.html"
       else "https://zynaddsubfx.sourceforge.io";
 
-    license = licenses.gpl2;
+    license = licenses.gpl2Plus;
     maintainers = with maintainers; [ goibhniu kira-bruneau ];
-    platforms = platforms.linux;
+    platforms = platforms.all;
+
+    # On macOS:
+    # - Tests don't compile (ld: unknown option: --no-as-needed)
+    # - ZynAddSubFX LV2 & VST plugin fail to compile (not setup to use ObjC version of pugl)
+    # - TTL generation crashes (`pointer being freed was not allocated`) for all VST plugins using AbstractFX
+    # - Zest UI fails to start on pulg_setup: Could not open display, aborting.
+    broken = stdenv.isDarwin;
```

Comment:
I went down a huge rabbit hole trying to get this working on Darwin. Turns out that the Darwin build is completely broken upstream, it looks like the latest version they have released as a binary is 3.0.3r20.

There's a bunch of things I fixed already, but it's not all the way there yet, and definitely shouldn't block the update (it wasn't building on Darwin before this change anyway).